riotfile.py

```diff
@@ -1120,6 +1120,45 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             },
             command="pytest {cmdargs} tests/contrib/aiopg",
         ),
+        Venv(
+            name="aiohttp",
+            command="pytest {cmdargs} tests/contrib/aiohttp",
+            pkgs={"pytest": [">=3"], "pytest-aiohttp": [latest]},
+            venvs=[
+                Venv(
+                    pys=["3.5", "3.6"],
+                    pkgs={
+                        "aiohttp": ["~=2.0", "~=2.1", "~=2.2"],
+                        "aiohttp_jinja": ["~=0.1.2", "~=0.1.3"],
+                        "yarl": "~=0.18.0",
+                    },
+                ),
+                Venv(
+                    pys=["3.5", "3.6", "3.7", "3.8"],
+                    pkgs={
+                        "aiohttp": "~=2.3",
+                        "aiohttp_jinja": "~=0.1.5",
+                        "yarl": "~=1.0",
+                    },
+                ),
+                Venv(
+                    pys=["3.5", "3.6", "3.7"],
+                    pkgs={
+                        "aiohttp": ["~=3.0.0", "~=3.1.0", "~=3.2.0", "~=3.3.0", "~=3.5.0", "~=3.6.0"],
+                        "aiohttp_jinja2": "~=0.15.0",
+                        "yarl": "~=1.0",
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.8"),
+                    pkgs={
+                        "aiohttp": ["~=3.0.0", "~=3.1.0", "~=3.2.0", "~=3.3.0", "~=3.6.0", latest],
```

Comment:
we are also testing these versions above for the 3.5, 3.6, 3.7 venv. does that mean you can just use `select_pys(min_version="3.5")` above and remove this venv?

Reply:
kyle also mentioned we don't support `latest` for `aiohttp`, so that could be why CI is failing